requirements/docs.txt

```diff
@@ -1,7 +1,7 @@
 sphinx>=4.0
-recommonmark  # fails with badges
-m2r  # fails with multi-line text
-nbsphinx>=0.8
+myst-parser
+nbsphinx>=0.8.5
+ipython[notebook]
```

Comment:
Wouldn't these be requirements inside the submodule?

Reply:
well yes but the submodules has also some basic ones and so you cannot simply link it inside with `-r ../notebooks/docs/requirements.txt` as it will cry that some are duplicate such as sphinx :incoming_envelope: